### DISCUSSION TITLE
Feature request: add a `isRefined` return value to `useRange` hook

### DISCUSSION THREAD

**Discussion by kirkwat** [algolia/instantsearch#6168] [Ideas] [answered]
I am using the `useRange` hook to create a component that allows the user to select from a list of ranges. My current implementation works as expected until I try to incorporate a `ClearRefinements` button. With `useRange`, I am unable to detect when the `ClearRefinements` button triggers the reset using just the `start` and `range` return values. Would it be possible to add an `isRefined` return value similar to the items in a refinement list? This would allow me to detect when the useRange filter is reset so I can show that a range is no longer selected. Or do you recommend a different approach for making a list of selectable ranges?

**Comment by Haroenv** [comment, marked as answer]
I think you actually want useNumericMenu for a list of ranges, which does have isRefined already built in. 

Regardless of that, when would isRefined be true? if the `min` or `max` value have been set of the range? That could make sense, although apart from your use case which fits other widgets better, I'm not sure what the goal would be.

Hope that helps!

**Comment by kirkwat** [comment]
Thank you for the help. I somehow missed the `useNumericalMenu` hook when looking over the docs and that was exactly what I needed. Ended up trying to recreate it with `useRange` which was an oversight, so this feature request is not necessary.
